.eslintrc.json

```diff
@@ -102,7 +102,7 @@
                     "error",
                     {
                         "mustMatch": "Copyright [0-9]{0,4} Adobe. All rights reserved.",
-                        "templateFile": "../../../config/license.js"
+                        "templateFile": "config/HEADER.js"
```

Comment:
I see that this is passing in CI, but locally, I'm still getting a lint error:

<img width="976" height="258" alt="Image" src="https://github.com/user-attachments/assets/82a2ebc0-0bd4-4908-a4c1-c88a8681a5b7" />

If I change this to:


```suggestion
                        "templateFile": "../config/HEADER.js"
```

I can get it to run and it finds lint errors:

<img width="1060" height="513" alt="Image" src="https://github.com/user-attachments/assets/27c28524-0164-4c2b-aa28-8659675e134d" />

Should we address these?